### PR TITLE
BAN HAMMER

### PR DESCRIPTION
**Pull request by LotuxPunk**
Added an option to ban the player.
Kick make no sense, because disconnect is already a sort of kick. 🗡 

I'll put another feature tomorrow,

### CODE REVIEW

File: src/main/java/me/fril/modblocker/MBConfig.java [hunk 1]
```diff
@@ -14,4 +14,12 @@
...
 	@Config.Comment("Here you put what message the client recieves when kicked from the server, use '@MOD' to represent the offending modid")
 	public String disconnectMessage = "Mod: &s is not allowed on this server.";
+
+	@Config.Name("Ban message")
+	@Config.Comment("Here you put what message the client recieves when banned from the server")
+	public String banMessage = ""; //Todo : Add a message
+
+	@Config.Name("Action")
+	@Config.Comment("What to do if one blacklisted mod is detected ? [1 : disconnect, 2: BAN HAMMER]")
+	public int actionID = 1;
```
Comment (Jeryn99) [inline src/main/java/me/fril/modblocker/MBConfig.java]: You can actually make these the literal enums

Have a example https://github.com/Suffril/Weeping-Angels/blob/abcdd11b4fed519452e54adea0a38a890568e37b/src/main/java/me/fril/angels/config/WAConfig.java#L63-L66

File: src/main/java/me/fril/modblocker/ModBlocker.java [hunk 2]
```diff
@@ -48,5 +52,10 @@ public static void handleDisconnect(EntityPlayerMP entityPlayerMP, String modid)
         String disconnectMessage = MBConfig.config.disconnectMessage.replaceAll("@MOD", modid);
         entityPlayerMP.connection.disconnect(new TextComponentString(disconnectMessage));
     }
+
+    public static void handleBan(EntityPlayerMP entityPlayerMP, String modid){
+        UserListBansEntry userlistbansentry = new UserListBansEntry(entityPlayerMP.getGameProfile(), (Date)null, MODID, (Date)null, MBConfig.config.banMessage);
```
Comment (Jeryn99) [inline src/main/java/me/fril/modblocker/ModBlocker.java:56]: Have you tested these values being null?

File: src/main/java/me/fril/modblocker/MBConfig.java (hunk 1 above)
Comment (LotuxPunk) [reply src/main/java/me/fril/modblocker/MBConfig.java]: Nice, I'll look at that

File: src/main/java/me/fril/modblocker/ModBlocker.java (hunk 2 above)
Comment (LotuxPunk) [reply src/main/java/me/fril/modblocker/ModBlocker.java:56]: Check CommandBanPlayer from Minecraft, it's almost the same xD

Comment (LotuxPunk) [reply src/main/java/me/fril/modblocker/ModBlocker.java:56]: But yes, I can fill these parameters correctly, let me handles that

Comment (LotuxPunk) [reply src/main/java/me/fril/modblocker/ModBlocker.java:56]: null is just here to put the ban permanent I think

Comment (Jeryn99) [reply src/main/java/me/fril/modblocker/ModBlocker.java:56]: Alright, cool cool

Comment (LotuxPunk) [reply src/main/java/me/fril/modblocker/ModBlocker.java:56]: then we could push the thing and define a ban duration

**Pull request merged**